test/tests/meshgenerators/patterned_mesh_generator/patterned_mesh_generator.i

```diff
@@ -28,9 +28,8 @@
     right_boundary_id = 2
     top_boundary_id = 3
     left_boundary_id = 4
-    y_width = 1.2
-    x_width = 1.2
-    #parallel_type = replicated
+    #y_width = 1.2
+    #x_width = 1.2
```

Comment:
Make sure you remove these completely rather than comment them out. It can be confusing for somebody less familiar with MOOSE to see unused (commented out) parameters in the tests.